jenkins.py

```diff
@@ -549,7 +550,13 @@ def read_metrics(module_config):
             else:
                 last_timestamp = int(time.time() * 1000) - (60 * 1000)
                 module_config["jobs_last_timestamp"][job["name"]] = last_timestamp
-            job_url = module_config["base_url"][:-1] + urllib.parse.urlparse(job["url"]).path
+            # If the Jenkins instance is configured with Jenkins URL, the Job URL will include the prefix
+            # Latest Jenkins versions quote job urls, unquoting as the _api_call func quotes urls
+            job_path = urllib.parse.urlparse(job["url"]).path
+            if module_config["path"] and job_path.startswith(module_config["path"]):
+                job_url = urllib_unquote(job["url"])
+            else:
+                job_url = module_config["base_url"][:-1] + urllib_unquote(urllib.parse.urlparse(job["url"]).path)
```

Comment:
```suggestion
                job_url = module_config["base_url"][:-1] + urllib_unquote(job_path)
```

Reply:
oh... that's an oversight 🤦  good catch!